### PR TITLE
[Feature2] file2 is added 

### PR DESCRIPTION
**Pull request by qaziaqibnawaz**
please approve the pull request

### CODE REVIEW

**Comment by Anup9661** [comment]
Good job

File: file2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+file2 made in local barch 2
```
Comment (Anup9661) [inline file2.txt:1]: give specific line

Comment (qaziaqibnawaz) [reply file2.txt:1]: OK SIR sure

**Review by Anup9661 — APPROVED** [review]
good

**Pull request merged**